### ISSUE TITLE
Well done!

### ISSUE THREAD

**Issue by Hvass-Labs** [KGSands/DRL-Chatbot#1]
I saw your link on youtube and I've taken a look at your project. It looks like it is a very well-crafted piece of software you have made - I like the polished code with detailed comments! I hope you get a good grade for this! :-D

**Comment by KGSands** [comment]
Thank you very much, I must admit I was quite nervous to send it to you! I can say with 100% certainty that I couldn't have done it in this timeframe without your brilliant video. I do owe you a small token of appreciation some time down the line, but for now, I'm still a poor student! 

With regards to the commenting - you'd set a good example on that one. Don't worry, I was totally pleased with the mark I achieved for this. :-D

**Comment by Hvass-Labs** [comment]
Really? Maybe I sound a little rough sometimes :-) But my philosophy is to criticize upwards in the hierarchy. I chew out the Google engineers because they are supposed to be the world's elite, but instead they set a horrible example with completely incomprehensible code that others then think is the acceptable norm of software engineering, while in reality it is creating huge problems for everyone who is going to build on that work in the future.

Your work is polished and beautiful. You explain in plain English everything you are doing in the code. I am very pleased if my work has inspired you to set such a high standard for yourself!

You did a great job and I hope you got a high grade for it!

I don't take donations anymore - I got so little anyway. But I always encourage people to e.g. donate an old but working PC to an orphanage or something like that, if they want to repay me for my work.

I wish you all the best in the future!

**Comment by KGSands** [comment]
Oh don't worry, I was just not sure if my code was worth posting - but it
seems after you "starred" it a few others did so too which was nice!

I totally understand where you are coming from though, it was difficult to
find other well-commented Reinforcement Learning examples - your code was
definitely an inspiration for me. My score was within the highest possible
grade boundary for this particular project.

I'm actually about to move country so maybe donating my PC is a decent idea
actually.

All the best to you too and thanks again!
